### PR TITLE
Coq Library of Undecidability Proofs 1.0.0

### PR DESCRIPTION
**Pull request by yforster**
An opam package for https://github.com/uds-psl/coq-library-undecidability/

I'm not entirely sure it's okay to have a `dev` package in `extra-dev` which however depends on `Coq 8.12`. If that's not the right place, please let me know.

### CODE REVIEW

**Comment by palmskog** [comment]
@yforster it's fine to have a dev package that depends on the regular Coq 8.12. However, the convention for version names is to use the scheme `8.12.dev` instead of `dev+8.12`. One important reason is that a pure `dev` version should be sorted *after* the 8.12 dev version. So, can you say for sure that you won't have a general `dev` package that tracks Coq `master`?

File: extra-dev/packages/coq-library-undecidability/coq-library-undecidability.dev+8.12/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          "Dominique Larchey-Wendling"
+          "Andrej Dudenhefner"
+          "Edith Heiter"
+          "Dominik Kirst"
+          "Fabian Kunze"
+          "Gert Smolka"
+          "Simon Spies"
+          "Dominik Wehr"
+          "Maximilian Wuttke"]
+license: "CeCILL"
```
Comment (palmskog) [inline extra-dev/packages/coq-library-undecidability/coq-library-undecidability.dev+8.12/opam:17]: Please specify the version of CECILL you're using. If it's version 2.1, then this should be `CECILL-2.1`. Same for the release.

Comment (yforster) [reply extra-dev/packages/coq-library-undecidability/coq-library-undecidability.dev+8.12/opam]: Fixed!

**Comment by yforster** [comment]
I fixed the CeCILL license version number and moved the package to `8.12.dev` - we don't plan on having a package tracking Coq `master`, but it's also not impossible, so better follow the convention.

**Pull request merged**